### PR TITLE
feat: Android dont create root keys for all networks adjustements

### PR DESCRIPTION
**Pull request by Dmitry-Borodin**
## Purpose
create proper path keys only for networks use is selected when keyset is created or restored

## Screenshots
| new white | new black |
|-|-|
| image | image |
| image | image |
| image | image |
| image | image |
|  |  |
| old black | old white |
|-|-|
| dark | light |
| Screenshot_20230608_173628 |  |
|  |  |

### CODE REVIEW

File: android/src/main/java/io/parity/signer/domain/Models.kt [hunk 1]
```diff
@@ -263,6 +263,9 @@ data class NetworkModel(
 	}
 }
 
+fun NetworkModel.createDefaultPath(): String = "//${this.title}"
```
Comment (Dmitry-Borodin) [inline android/src/main/java/io/parity/signer/domain/Models.kt:266]: @krodak are you using the same logic on iOS?

**Comment by Dmitry-Borodin** [comment]
I will reopen this when will use new rust functions instead of navigation or just new logic for default network path

File: android/src/main/java/io/parity/signer/domain/Models.kt (hunk 1 above)
Comment (krodak) [reply android/src/main/java/io/parity/signer/domain/Models.kt:266]:
```swift
 let pathAndNetworks: [(path: String, network: MmNetwork)] = networks
            .map { (path: "//\($0.title)", network: $0) }
```

Comment (prybalko) [reply android/src/main/java/io/parity/signer/domain/Models.kt:266]: `MNetworkDetails` model as a `path_id` and that's exactly the derivation path prefix.

**Review by tetekinandrey — APPROVED** [review]

**Pull request merged**